Paco-Server/src/com/google/sampling/experiential/shared/Views.java

```diff
@@ -1,4 +1,4 @@
-package com.google.sampling.experiential.model;
+package com.google.sampling.experiential.shared;
```

Comment:
Sorry, I meant moving this to the git/paco/Shared/src/... project and under model2 actually (where JsonConverter lives).